QUEST_LV_0300.tsv

```diff
@@ -143,10 +143,10 @@ QUEST_LV_0300_20150729_000142	Now you've collected them all. Here, I'll give you
 QUEST_LV_0300_20150729_000143	I just want to rest...
 QUEST_LV_0300_20150729_000144	Is this the last time? Really.
 QUEST_LV_0300_20150729_000145	Let's see if we have all the detonators.
-QUEST_LV_0300_20150729_000146	You've came here... Goodbye for now.
+QUEST_LV_0300_20150729_000146	You've come here...goodbye, for now.
```

Comment:
The line is actually past tense.

You've came... but it's already time for me to go.